kore/src/Kore/Exec.hs

```diff
@@ -567,7 +567,7 @@ assertSomeClaims claims =
 makeClaim
     :: Goal.FromRulePattern claim
     => Goal.ToRulePattern claim
-    => (Attribute.Axiom Symbol, claim) -> claim
+    => (Attribute.Axiom Symbol Variable, claim) -> claim
```

Comment:
I left this a `Variable`:
`fromRulePattern` has to return a `RulePattern Variable` because it's first parameter (`ruleType`) has that type, returned by `toRulePattern`